app/controllers/vendor_api/applications_controller.rb

```diff
@@ -22,7 +22,26 @@ def serialized_application_choices_data
     end
 
     def get_application_choices_for_provider_since(since:)
-      application_choices_visible_to_provider
+      application_choices_visible_to_provider(
+        [
```

Comment:
But what it confused me yesterday it is because this class also has the includes https://github.com/DFE-Digital/apply-for-teacher-training/blob/a459c8551548db7180ddc704389c78e2783945da/app/queries/get_application_choices_for_providers.rb#L2
And I was in doubt where to put the includes. But happy to move to the application data concerns.

Reply:
Done! ♻️